controller/nodelabel_controller.go

```diff
@@ -54,11 +52,11 @@ func (r *ReconcileNodeLabel) Reconcile(req reconcile.Request) (reconcile.Result,
 	log := r.Log.WithValues("node-label-operator", req.NamespacedName)
 
 	var configMap corev1.ConfigMap
-	optionsNamespacedName := OptionsConfigMapNamespacedName() // assuming "node-label-operator" and "node-label-operator-system", is this okay
+	optionsNamespacedName := options.OptionsConfigMapNamespacedName() // assuming "node-label-operator" and "node-label-operator-system", is this okay
```

Comment:
`options.ConfigMapNamespaceName()` remove the Options prefix as it is now in the options package